### PR TITLE
Centos test

### PR DESCRIPTION
**Pull request by devidask27**
Test cases added for prerequisite check for check-node for both versions Centos and Debian

### CODE REVIEW

**Review by sahil-lakhwani — CHANGES_REQUESTED** [review]
I think we can also write tests the `err` field in the  `want` struct. For this, have the executor return an error.

File: pkg/platform/centos/centos_test.go [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+			},
+			want: want{
+				result: true,
+			},
+		},
+		"CheckFail": {
+			args: args{
+				exec: &cmdexec.MockExecutor{
+					MockRunWithStdout: func(name string, args ...string) (string, error) {
+						return "1", nil
```
Comment (anupbarve) [inline pkg/platform/centos/centos_test.go:40]: Could you please state as to why this should fail ?
From reading the test case, we wouldn't know what are success and failure conditions.
Put appropriate comments for both success and failure conditions.

This applies to all the test cases.

**Comment by anupbarve** [comment]
Also run a TC build with your branch and post the TC build link here.

File: pkg/platform/centos/centos_test.go (hunk 1 above)
Comment (devidask27) [reply pkg/platform/centos/centos_test.go:40]: Sure i will do it.

File: pkg/platform/centos/centos_test.go [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+			}
+			if diff := cmp.Diff(tc.want.result, o); diff != "" {
+				t.Errorf("r: -want, +got:\n%s", diff)
+			}
+		})
+	}
+}
+
+//Port check test case
+func TestPort(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (devidask27) [reply pkg/platform/centos/centos_test.go:239]: Tested with mixed output

File: pkg/platform/centos/centos_test.go (hunk 1 above)
Comment (devidask27) [reply pkg/platform/centos/centos_test.go:44]: Added comments for each test case

File: pkg/platform/centos/centos_test.go [hunk 3]
```diff
@@ -21,6 +21,8 @@ func TestCPU(t *testing.T) {
 		args
 		want
 	}{
+		//Success case. Minimun CPU required is 2
+		//Returning 4 CPUS. Therfore test case should pass
```
Comment (anupbarve) [inline pkg/platform/centos/centos_test.go:25]: nit: Therfore spelling here and everywhere.

**Review by anupbarve — CHANGES_REQUESTED** [review]
Could you please also add test cases for `removePyCli` functionality.

**Pull request closed without merging**